source/module_ri/exx_abfs-construct_orbs.cpp

```diff
@@ -44,6 +44,17 @@ std::vector<std::vector<std::vector<Numerical_Orbital_Lm>>> Exx_Abfs::Construct_
 			}
 		}
 	}
+
+	for (int T = 0;  T < orbs.size() ; T++)
+	{
+		for (int L=orbs[T].size()-1; L >= 0  ; L--)
+		{
+			if (orbs[T][L].size()>0)
```

Comment:
This code looks confused to me, it looks only the L = orbs[T].size()-1 is used for the judgement, then break the loop or the orbs[T] will be resized in the L loop?

Reply:
If `orbs[T] = 2s 2p 1d 0f 0g`, it will transformed to `2s 2p 1d`